ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node/TPraos.hs

```diff
@@ -377,8 +380,10 @@ registerGenesisStaking staking nes = nes {
     -- See STS DELEG for details
     dState' :: SL.DState (EraCrypto era)
     dState' = (SL.dpsDState dpState) {
-          SL._unified = UM.unify
-            ( Map.map (const $ SL.Coin 0)
+          SL.dsUnified = UM.unify
```

Comment:
Removed the TODO. From our conversation on slack this is not important, since it is only used during benchmarking, so keeping it at arbitrary value `0` is just fine.